volumes/miovision/sql/table/create-table-volumes_15min_mvt.sql

```diff
@@ -9,40 +9,43 @@
     volume numeric,
     processed boolean,
     CONSTRAINT volumes_15min_mvt_int_uid_dt_bin_class_leg_mvmt_uid_pkey
-    PRIMARY KEY (intersection_uid, datetime_bin, classification_uid, leg, movement_uid)
+    PRIMARY KEY (
+        volume_15min_mvt_uid, intersection_uid, datetime_bin, classification_uid, leg, movement_uid
+    )
```

Comment:
This addition of `volume_15min_mvt_uid` to the primary key was already applied in #735 but somehow not reflected in that PR.